### PR TITLE
[Dicom-Cast] ARM template for table storage and partial validation

### PR DESCRIPTION
**Pull request by abdullah248**
## Description
Update ARM template to configure partial validation and storage to table storage.

## Related issues
[AB#78974](https://microsofthealth.visualstudio.com/f8da5110-49b1-4e9f-9022-2f58b6124ff9/_workitems/edit/78974)

## Testing
Created deployment with the template.

### CODE REVIEW

File: converter/dicom-cast/samples/templates/default-azuredeploy.json [hunk 1]
```diff
@@ -94,6 +94,20 @@
             "metadata": {
                 "description": "The endpoint of the FHIR server."
             }
+        },
+        "enableTableStorage": {
```
Comment (smithago) [inline converter/dicom-cast/samples/templates/default-azuredeploy.json:98]: Do we want to expose these has params? Or should we just have the default behavior of, enableTableStorage: True and set the IgnoreSyncOfInvalidTags: true in the container.

Comment (smithago) [reply converter/dicom-cast/samples/templates/default-azuredeploy.json:98]: Basically reduce the settings here to get the default dicom-cast deployment.

Comment (abdullah248) [reply converter/dicom-cast/samples/templates/default-azuredeploy.json]: Updated to set tablestorage to true and leave IgnoreSyncOfInvalidTags as configurable

File: converter/dicom-cast/samples/templates/default-azuredeploy.json [hunk 2]
```diff
@@ -94,6 +94,13 @@
             "metadata": {
                 "description": "The endpoint of the FHIR server."
             }
+        },
+        "IgnoreSyncOfInvalidTagValue": {
+            "defaultValue": false,
```
Comment (smithago) [inline converter/dicom-cast/samples/templates/default-azuredeploy.json:99]: Sorry Abdullah, should we make this true by default?

**Review by smithago — APPROVED** [review]

File: converter/dicom-cast/samples/templates/default-azuredeploy.json (hunk 2 above)
Comment (abdullah248) [reply converter/dicom-cast/samples/templates/default-azuredeploy.json]: Updated

**Pull request merged**